sound/soc/intel/boards/cml_rt1011_rt5682.c

```diff
@@ -547,7 +547,7 @@ static int snd_cml_rt1011_probe(struct platform_device *pdev)
 	if (sof_rt1011_quirk & (SOF_RT1011_SPEAKER_TL |
 				SOF_RT1011_SPEAKER_TR)) {
 		for_each_card_prelinks(&snd_soc_card_cml, i, dai_link) {
-			if (!strcmp(cml_rt1011_rt5682_dailink[i].codecs->dai_name,
+			if (!strcmp(cml_rt1011_rt5682_dailink[i].codecs[0].dai_name,
```

Comment:
@fredoh9 ``cml_rt1011_rt5682_dailink[i].codecs[0].dai_name`` should be ``dai_link->codecs[0].dai_name``. That is what for_each_card_prelinks() for. Sorry I didn't notice that when reviewing https://github.com/thesofproject/linux/pull/2173/commits/062af53735f5bbfdde1bf32234e1b2971ec6a7ec. Can you add a fixup commit for 062af53735f5bbfdde1bf32234e1b2971ec6a7ec?
To prevent conflict, we should fixup 062af53735f5bbfdde1bf32234e1b2971ec6a7ec first, then this commit.

Reply:
Sorry, I missed that too. Will update this shortly.